orm/encoding/ormfield/bytes_test.go

```diff
@@ -0,0 +1,137 @@
+package ormfield_test
+
+import (
+	"bytes"
+	"testing"
+
+	"cosmossdk.io/orm/encoding/ormfield"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+// MockReader implements the Reader interface for testing
+type MockReader struct {
+	*bytes.Reader
+}
+
+func NewMockReader(data []byte) *MockReader {
```

Comment:
Why this new definition and type just to avoid typing bytes.NewReader? You simply could write
`var newMockReader = bytes.NewReader`